app/assets/javascripts/admin/views/help/faq_view.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from "react";
+
+const FAQView = () => (
+  <div className="container help">
+    <h3>FAQ</h3>
+    <p />
+    <h4>Which browsers are supported for webKnossos?</h4>
+    <p>
+      webKnossos runs best with the latest version of Google Chrome. We use it to test and develop
+      the software and recommend it.
+    </p>
+    <p>
+      However, any other modern browser in their latest version should work as well. We have had
+      good success with the following browsers:
+    </p>
+    <ul>
+      <li>Firefox 51+</li>
+      <li>Opera 43+</li>
+      <li>Edge 14+</li>
+      <li>Safari 10</li>
+    </ul>
+    <p>Internet Explorer is not supported.</p>
+    <h4>What are possible task queries?</h4>
```

Comment:
Either the master was not merged for some time or this got lost during the merge, but I remember that you removed the task query explanations and rewrote the supported browsers part in another PR that was already merged.

Reply:
Merge error :-) Ups.